company/update.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to update your company's profile and settings.
+---
+
+# Update company profile and settings
+
+Name and roles cannot be updated.
+
+Update your profile and settings:
+
+## Request
+
+{% api-method method="put" host="https://api.accp.tradecloud1.com/v2" path="/company/:id"%} 
+{% api-method-summary %} Update company by id {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Bearer Access-Token {% endapi-method-parameter %} 
+{% endapi-method-headers %}
+{% api-method-path-parameters %} 
+{% api-method-parameter name="id" type="uuid" required=true %} ID company to update. {% endapi-method-parameter %}
+{% endapi-method-path-parameters %}
+{% endapi-method-request %}
+{% endapi-method-spec %}
+{% endapi-method %}
+
+### Body
+
+In the request body you may update:
+
+* profile fields
+* integration settings, either webhook or ftp
+* copy unchanged fields from [Get company profile and settings](get.md)
```

Comment:
Is this a requirement? If so, we should make this clearer I think.

Reply:
the PUT will overwrite the company object (well I assuming this is the current behavior, but is it desired that is missing field is nullified?)
I improved the bullet